src/Fixer/PhpUnit/PhpUnitExpectationFixer.php

```diff
@@ -0,0 +1,255 @@
+            }
+
+            $openIndex = $tokens->getNextTokenOfKind($index, ['(']);
+            $closeIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $openIndex);
+
+            $arguments = $argumentsAnalyzer->getArguments($tokens, $openIndex, $closeIndex);
+            $argumentsCnt = count($arguments);
+
+            $argumentsReplacements = ['expectException', $this->methodMap[$tokens[$index]->getContent()], 'expectExceptionCode'];
+
+            $indent = "\n".$this->detectIndent($tokens, $thisIndex); // TODO whitespaceaware shit
+
+            $isMultilineWhitespace = false;
+
+            for ($cnt = $argumentsCnt - 1; $cnt >= 1; --$cnt) {
+                $argStart = array_keys($arguments)[$cnt];
+                $argBefore = $tokens->getPrevMeaningfulToken($argStart);
+                $isMultilineWhitespace = $isMultilineWhitespace || ($tokens[$argStart]->isWhitespace() && !$tokens[$argStart]->isWhitespace(" \t"));
+
+                $tokensOverrideArgStart = [
+                    new Token([T_WHITESPACE, $indent]),
+                    new Token([T_VARIABLE, '$this']),
+                    new Token([T_OBJECT_OPERATOR, '->']),
+                    new Token([T_STRING, $argumentsReplacements[$cnt]]),
+                    new Token('('),
+                ];
+                $tokensOverrideArgBefore = [
+                    new Token(')'),
+                    new Token(';'),
+                ];
+
+                if ($isMultilineWhitespace) {
+                    array_push($tokensOverrideArgStart, new Token([T_WHITESPACE, $indent.'    '])); // TODO whitespaceaware shit
+                    array_unshift($tokensOverrideArgBefore, new Token([T_WHITESPACE, $indent]));
+                }
+
+                if ($tokens[$argStart]->isWhitespace()) {
+                    $tokens->overrideRange($argStart, $argStart, $tokensOverrideArgStart);
+                } else {
+                    $tokens->insertAt($argStart, $tokensOverrideArgStart);
+                }
+
+                $tokens->overrideRange($argBefore, $argBefore, $tokensOverrideArgBefore);
+
+                $limit = $tokens->count();
+            }
+
+            $tokens[$index] = new Token([T_STRING, 'expectException']);
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function createConfigurationDefinition()
+    {
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('target', 'Target version of PHPUnit.'))
+                ->setAllowedTypes(['string'])
+                ->setAllowedValues([PhpUnitTargetVersion::VERSION_5_2, PhpUnitTargetVersion::VERSION_5_6, PhpUnitTargetVersion::VERSION_NEWEST])
+                ->setDefault(PhpUnitTargetVersion::VERSION_NEWEST)
+                ->getOption(),
+        ]);
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     *
+     * @return string
+     */
+    private function detectIndent(Tokens $tokens, $index)
```

Comment:
Looks like this is used in a couple of places, maybe pull up?

Reply:
it's in couple of places already on master, let us not do this refactoring here, PR be big anyway